src/objective-c/GRPCClient/private/GRPCSecureChannel.m

```diff
@@ -33,12 +33,24 @@
 
 #import "GRPCSecureChannel.h"
 
-#import <grpc/grpc_security.h>
+grpc_credentials *CertificatesAtPath(NSString *path) {
```

Comment:
This should probably be allowed to be `nil`, so that we can use the default SSL root certs location. Then, of course, we'll have to make sure we can distinguish between creating a channel with a `nil` cert path and creating an insecure channel.